vars/ostreeImageBootSanity.groovy

```diff
@@ -62,7 +62,7 @@ def call(body) {
                               "export JENKINS_BUILD_TAG=\"${BUILD_TAG}-${current_stage}\"\n" +
                               "export OSTREE_BRANCH=\"${OSTREE_BRANCH}\"\n" +
                               "export ANSIBLE_HOST_KEY_CHECKING=\"False\"\n"
-                pipelineUtils.rsyncResults(current_stage, 'duffy-key')
+                pipelineUtils.runTaskAndReturnLogs(current_stage, 'duffy-key')
```

Comment:
I assume you are calling this other function here but in the monolithic Jenkinsfile-rf-pipeline you are calling rsyncResults instead is that the idea?

Reply:
correct